src/modules/jcms_admin/src/HtmlMarkdownSerializer.php

```diff
@@ -100,7 +100,7 @@ private function preserveOutput(string $html, array $context = []) : string {
       }
     }
     $preserve = implode('', $lines);
-    return preg_replace_callback('~' . $bc . '(code|table)[^' . $bc . ']*' . $bc . '([^' . $bc . ']*)' . $bc . '/\1' . $bc . '~s', function ($matches) use ($bc, $encode) {
```

Comment:
can't understand why `$bc` was `ø`, but ok

Reply:
It was just an arbitrary character.